crawl-ref/source/dat/descript/quotes.txt

```diff
@@ -2037,6 +2037,15 @@ meliai
  boundless earth.”
     -Hesiod, _Theogony_, 182-187, trans. H.G. Evelyn-White. 1920
 %%%%
+Mephitic Cloud spell
+
+"For the past few years Asiatic cholera had shown a strong tendency to spread.
+ Its source was the hot, moist swamps of the delta of the Ganges, where it bred
+ in the mephitic air of that primeval island-jungle, among whose bamboo thickets
+ the tiger crouches, where life of every sort flourishes in rankest abundance,
+ and only man avoids the spot."
+    -Thomas Mann, _Diaries_.
```

Comment:
For quotes we prefer the quote in the original language, or if translated cite the translation you're using as well. For Mann's *Diaries*, if you're citing the English edition I think there is only one translation out there (I didn't spend too long with the library database). If you decide to find the German I think there is both the unedited transcription and an abridged version, so do specify the correct source.

Reply:
Turns out I misunderstood the quotation source and it's actually from "Death in Venice", which is more satisfying anway. I'll update.
Question about the language. I have the German version. Are you suggesting to include this in the English game?
Should it come along with a translation?

Reply:
Place it in the original German, without a translation. We have quotes in Latin ;)